presto-docs/src/main/sphinx/release/release-0.276.rst

```diff
@@ -0,0 +1,60 @@
+=============
+Release 0.276
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where a query with an order by clause and an unpartitioned window function sometimes returned unordered results.
+* Improve Graphviz output for JOIN nodes.
+* Improve graphviz output with estimate stats.
+* Add sized-based split weights for Hudi connector to improve query performance. This is enabled by default, controlled by configuration property ``hudi.size-based-split-weights-enabled`` and session property ``hudi.size_based_split_weights_enabled``. Two more configuration properties are added to adjust the weight: ``hudi.standard-split-weight-size`` to configure the split size corresponding to the standard split weight, and ``hudi.minimum-assigned-split-weight`` to configure the minimum split weight.
+* Add support for complex JsonPath expressions in JSON_EXTRACT, JSON_EXTRACT_SCALAR and JSON_SIZE using Jayway.
+* Add support to nested SQL functions with lambdas when ``inline_sql_functions = false``.
+* Added the types and SqlFunctions defined by GeoPlugin to `BuiltInTypeAndFunctionNamespaceManager.java` for registering.
+* Adding documentation for construct_tdigest.
+* Replace joda.time codes with java.time counterparts.
+* Remove joda.time dependency at presco-clickhouse.
+* Remove usage and package dependency of joda-time from presto-iceberg inline with JSR-310 suggestions to utilise java.time.* instead.
+* Removed presto-geospatial module.
+* Cover exception cases (DateTimeException) since joda.time does not throw exceptions.
+* Make query.max-total-memory-per-node property optional with default value dependent on query.max-memory-per-node.
+* Moved the relevant classes to presto-main to avoid too many circular dependency issues which would have occurred if movement was done to presto-common rather.
+* Moved the relevant test classes to presto-main and presto-memory.
+* Optimized the number of hdfsConfiguration copies being done by using copy-on-first-write technique. This feature is enabled by default and can be controlled by setting the ``hive.copy-on-first-write-configuration`` configuration property appropriately.
```

Comment:
This is an optimization and can move below Fixes.
See order of changes. : https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes